pkg/admission/prep_downscale.go

```diff
@@ -159,6 +159,22 @@ func prepareDownscale(ctx context.Context, logger log.Logger, ar v1.AdmissionRev
 				ar.Request.Resource.Resource, ar.Request.Name, ar.Request.Namespace, *oldReplicas, *newReplicas, foundSts.name, foundSts.lastDownscaleTime, foundSts.waitTime,
 			)
 		}
+
+		foundSts, err = findStatefulSetWithNonUpdatedReplicas(ctx, api, ar.Request.Namespace, rolloutGroup)
+		if err != nil {
+			level.Warn(logger).Log("msg", "downscale not allowed due to error while finding other statefulsets", "err", err)
```

Comment:
Should we log the same message that we're returning to the caller to help debugging?